etc/module.xml

```diff
@@ -24,6 +24,7 @@
             <module name="Magento_Sales"/>
             <module name="Magento_Checkout"/>
             <module name="Magento_Tax"/>
+            <module name="Magento_GiftCardAccount"/>
```

Comment:
What happens here if the module does not exist? Assuming it is ignored gracefully.

Reply:
Nothing, i checked. It just determines (or like) the order of module load.